rustls/tests/api.rs

```diff
@@ -34,7 +34,7 @@ use rustls::{
     crypto::aws_lc_rs::hpke::ALL_SUPPORTED_SUITES,
     internal::msgs::base::PayloadU16,
     internal::msgs::handshake::{
-        EchConfigPayload, EchConfigContents, HpkeKeyConfig, HpkeSymmetricCipherSuite,
+        EchConfigContents, EchConfigPayload, HpkeKeyConfig, HpkeSymmetricCipherSuite,
```

Comment:
Oops! thought I fixed the last of this stuff in https://github.com/rustls/rustls/pull/2002 but I forgot about the dance required to reformat `api.rs`. Thanks for fixing :+1: